splice_machine/src/main/java/com/splicemachine/derby/stream/iterator/GetNLJoinIterator.java

```diff
@@ -85,24 +83,15 @@ protected void init() throws Exception {
             cm = ContextService.getFactory().newContextManager();
```

Comment:
Creating a new ContextManager is unnecessary and undesirable here.
If somehow this code is still required, a proper fix needs to be implemented/extended at a higher level.

Reply:
Benchmark results after latest code push:
java -jar SpliceBenchmark.jar -S 10000 -c 1 -n 10000
```
##### INSERT vs. UPDATE Trigger Benchmark #####
2021-02-25 21:11:27    Setup
2021-02-25 21:11:30    Create and populate tables
2021-02-25 21:11:45    Statistics:
	CREATE	calls: 20000	time: 15 s	avg: 1 ms
2021-02-25 21:11:45    Analyze schema, create triggers
2021-02-25 21:11:53    Do inserts
2021-02-25 21:12:53    Statistics:
	INSERT_T	calls: 8074	time: 29 s	avg: 4 ms
	INSERT_NOT	calls: 8073	time: 30 s	avg: 4 ms
2021-02-25 21:13:08    Statistics:
	INSERT_T	calls: 1926	time: 6 s	avg: 4 ms
	INSERT_NOT	calls: 1927	time: 7 s	avg: 4 ms
2021-02-25 21:13:08    Do updates
2021-02-25 21:14:08    Statistics:
	UPDATE_T	calls: 4708	time: 31 s	avg: 7 ms
	UPDATE_NOT	calls: 4707	time: 29 s	avg: 6 ms
2021-02-25 21:15:08    Statistics:
	UPDATE_T	calls: 4891	time: 31 s	avg: 6 ms
	UPDATE_NOT	calls: 4891	time: 28 s	avg: 6 ms
2021-02-25 21:15:13    Statistics:
	UPDATE_T	calls: 401	time: 2 s	avg: 6 ms
	UPDATE_NOT	calls: 402	time: 2 s	avg: 6 ms
2021-02-25 21:15:13    Finished
```